docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -214,10 +214,11 @@ function TopLayoutBlog(props) {
   const { description, rendered, title, headers } = docs.en;
   const finalTitle = title || headers.title;
   const router = useRouter();
+  const slug = router.pathname.replace(/\/blog\//, '');
   const { canonicalAs } = pathnameToLanguage(router.asPath);
   const card =
     headers.card === 'true'
-      ? `https://mui.com/static${router.pathname}/card.png`
+      ? `https://mui.com/static/blog/${slug}/card.png`
```

Comment:
A small fix, only to help search in the codebase, for places here the images inside the blog folder are used.